src/math/FGFunction.cpp

```diff
@@ -527,6 +527,15 @@ void FGFunction::Load(Element* el, FGPropertyValue* var, FGFDMExec* fdmex,
                  return y != 0.0 ? fmod(p[0]->GetValue(), y) : HUGE_VAL;
                };
       Parameters.push_back(new aFunc<decltype(f), 2>(f, fdmex, element, Prefix, var));
+    } else if (operation == "roundmultiple") {
+      auto f = [](const decltype(Parameters)& p)->double {
+                 double multiple = p.size() == 1 ? 1.0 : p[1]->GetValue();
+                 return round((p[0]->GetValue() / multiple)) * multiple;
+               };
+      if (element->GetNumElements() == 1)
+        Parameters.push_back(new aFunc<decltype(f), 1>(f, fdmex, element, Prefix, var));
```

Comment:
If there is only one parameter just use plain `round()` instead of using the lambda `f` which divide/multiply by 1.0
```suggestion
        Parameters.push_back(make_MathFn(round, fdmex, element, Prefix, var));
```